README.md

```diff
@@ -365,12 +365,40 @@ If you wish to treat your enviroment variables as strings always, even if they a
 ## Installation
 
 - download one of [releases](https://github.com/roboll/helmfile/releases) or
-- run as a [container](https://quay.io/roboll/helmfile) or
+- [run as a container](#running-as-a-container) or
 - Archlinux: install via `pacman -S helmfile` or from [AUR](https://aur.archlinux.org/packages/kubernetes-helmfile-bin/) or
 - openSUSE: install via `zypper in helmfile` assuming you are on Tumbleweed; if you are on Leap you must add the [kubic](https://download.opensuse.org/repositories/devel:/kubic/) repo for your distribution version once before that command, e.g. `zypper ar https://download.opensuse.org/repositories/devel:/kubic/openSUSE_Leap_\$releasever kubic`, or
 - Windows (using [scoop](https://scoop.sh/)): `scoop install helmfile`
 - macOS (using [homebrew](https://brew.sh/)): `brew install helmfile`
 
+### Running as a container
+
+The [Helmfile Docker images are available in Quay](https://quay.io/roboll/helmfile). There is no `latest` tag, since the `0.x` versions can contain breaking changes, so make sure you pick the right tag. Example using `helmfile 0.135.0`:
+
+```sh-session
+# helm 2
+$ docker run --rm --net=host -v "${HOME}/.kube:/root/.kube" -v "${HOME}/.helm:/root/.helm" -v "${PWD}:/wd" quay.io/roboll/helmfile:v0.135.0 helmfile sync
```

Comment:
```suggestion
$ docker run --rm --net=host -v "${HOME}/.kube:/root/.kube" -v "${HOME}/.helm:/root/.helm" -v "${PWD}:/wd" --workdir /wd quay.io/roboll/helmfile:v0.135.0 helmfile sync
```

This does make sense, but as I've commented at https://github.com/roboll/helmfile/pull/1607/files#r540608083, I'd like to use `/` as the default workdir and explicitly set `--workdir /wd` for this "recommended" use-case.

I ABSOLUTELY agree we should have a great default, but I believe the image is already used widely in production and our users won't love breaking changes :)

Reply:
Makes sense, to prevent a breaking change. Perhaps in the future. :)